docker-compose.yaml

```diff
@@ -79,78 +80,48 @@ services:
     ports:
       - "5004:5004"
 
-  prometheus:
-    build:
-      context: ./prometheus
-      dockerfile: Dockerfile
-      args:
-        PROM_FISHJAM_TARGETS: ${PROM_FISHJAM_TARGETS:-fishjam1:9568,fishjam2:9568}
-        # Through 172.17.0.1 address docker containers (not running in network mode host) can access
-        # services running on host OS (localhost). In this example through this address prometheus could access
-        # node-exporter which is running in network mode host.
-        PROM_NODE_TARGETS: '172.17.0.1:9100'
-    container_name: prometheus
-    restart: on-failure
-    command:
-      - --config.file=/etc/prometheus/prometheus.yml
-      - --storage.tsdb.path=/prometheus
-      - '--web.console.libraries=/usr/share/prometheus/console_libraries'
-      - '--web.console.templates=/usr/share/prometheus/consoles'
-    volumes:
-      - prometheus_data:/prometheus
-    depends_on:
-      - fishjam1
-      - fishjam2
-
   grafana:
     build:
-      context: ./grafana
+      context: ./infra/grafana/
       dockerfile: Dockerfile
     container_name: grafana
-    restart: on-failure
-    volumes:
-      - grafana_data:/var/lib/grafana
-    depends_on:
-      - prometheus
-    environment:
-      GF_SECURITY_ADMIN_PASSWORD: "${GF_SECURITY_ADMIN_PASSWORD}"
-      GF_SECURITY_ADMIN_USER: "${GF_SECURITY_ADMIN_USER}"
     ports:
-      - '3000:3000'
-    expose:
-      - 3000
-
-  node-exporter:
-    image: prom/node-exporter:v1.6.1
-    container_name: node_exporter
-    restart: on-failure
-    network_mode: host
-    command:
-      - --path.rootfs=/host
+      - "3000:3000"
+    environment:
+      - GF_AUTH_ANONYMOUS_ENABLED=true
+      - GF_AUTH_ANONYMOUS_ORG_ROLE=Admin
     volumes:
-      - /:/host:ro,rslave
+      - grafana-storage:/var/lib/grafana
 
   loki:
     build:
-      context: ./loki
+      context: ./infra/loki/
       dockerfile: Dockerfile
+    container_name: loki
+    command: -config.file=/etc/loki/local-config.yaml
     volumes:
-      - loki_data:/loki
-    ports:
-      - "3100:3100"
+      - loki-data:/loki
 
-  promtail:
+  grafana-alloy:
     build:
-      context: ./promtail
+      context: ./infra/alloy/
       dockerfile: Dockerfile
-      args:
-        LOKI_ADDRESS: loki
-    volumes:
-      # to read container labels and logs
-      - /var/run/docker.sock:/var/run/docker.sock
-      - /var/lib/docker/containers:/var/lib/docker/containers
+    container_name: alloy
+    env_file:
+      - './.env.example'
+    entrypoint:
+      - '/bin/alloy'
+      - 'run'
+      - '--server.http.listen-addr=0.0.0.0:12345'
+      - '--config.extra-args="-config.expand-env"'
+      - '${ALLOY_CONFIG_PATH}/${ALLOY_CONFIG_FILE}'
+    expose:
+      - "8027"
```

Comment:
**nitpick:** `EXPOSE` is more of a comment, than a command, but it falsely suggests exposing ports that shouldn't be accessible to external networks.